src/components/PeopleFilters.tsx

```diff
@@ -1,17 +1,97 @@
-export const PeopleFilters = () => {
+import cn from 'classnames';
+import { SearchLink } from './SearchLink';
+import { FilterBy } from '../types/FilterBy';
+import { useEffect, useState } from 'react';
+import { centuryList } from '../utils/centuryList';
+
+type PeopleFilterProps = {
+  searchParams: URLSearchParams;
+  setQuery: React.Dispatch<React.SetStateAction<string>>;
+  setSearchParams: React.Dispatch<React.SetStateAction<URLSearchParams>>;
+  activeQuery: string;
+  toggleCenturies: (currentCentury: number) => string[];
+  activeCenturies: string[];
+};
+
+export const PeopleFilters = ({
+  toggleCenturies,
+  searchParams,
+  setQuery,
+  activeQuery,
+  setSearchParams,
+  activeCenturies,
+}: PeopleFilterProps) => {
+  const [activeFilter, setActiveFilter] = useState('');
+
+  useEffect(() => {
+    setActiveFilter(searchParams.get('sex') || '');
+    setQuery(searchParams.get('query') || '');
```

Comment:
you have the `enum QueryParams` so use it 

```suggestion
    setActiveFilter(searchParams.get(QueryParams.Sex) || '');
    setQuery(searchParams.get(QueryParams.Query) || '');
```